v3/plugins/vuejs/vetur/0.26.1/meta.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+publisher: vuejs
+name: vetur
+version: 0.26.1
+type: VS Code extension
+displayName: Vetur
+title: Rich Vue.js support
+description: Vue tooling for VS Code
+icon: https://upload.wikimedia.org/wikipedia/commons/9/95/Vue.js_Logo_2.svg
+repository: https://github.com/vuejs/vetur.git
+category: Language
+firstPublicationDate: "2020-08-26"
+spec:
+  containers:
+    - image: 'quay.io/eclipse/che-sidecar-node:10-e025a40'
```

Comment:
Can we use node 12? All the plugins/sidecars are using node 12 now, image is `quay.io/eclipse/che-sidecar-node:12-da9c36b`. Please also update the `vscode-extensions.json` file.